crates/matrix-sdk-ui/tests/integration/room_list_service.rs

```diff
@@ -537,6 +543,7 @@ async fn test_sync_resumes_from_previous_state() -> Result<(), Error> {
                 "lists": {
                     ALL_ROOMS: {
                         "ranges": [[0, 9]],
+                        "timeline_limit": 0,
```

Comment:
That is a bug, fixed in https://github.com/matrix-org/matrix-rust-sdk/pull/4039 :-).

Reply:
Ooops, I thought it was mocks not being fully fleshed out 🙈